bigtable-hbase/src/main/java/org/apache/hadoop/hbase/client/AbstractBigtableConnection.java

```diff
@@ -105,6 +105,20 @@ public AbstractBigtableConnection(Configuration conf) throws IOException {
     this(conf, false, null, null);
   }
 
+  /**
+   * Default constructor called from {@link ConnectionFactory#createConnection(Configuration)} and
+   * in its many forms via reflection.
+   * 
+   * @param conf The configuration of 
+   * @param managed This should always be false. It's an artifact of old HBase behavior.
```

Comment:
Unfortunately, not.  ConnectionFactory.createConnection(Configuration) uses reflection and looks for this signature.